oauthproxy.go

```diff
@@ -626,13 +626,6 @@ func (p *OAuthProxy) isTrustedIP(req *http.Request) bool {
 // SignInPage writes the sign in template to the response
 func (p *OAuthProxy) SignInPage(rw http.ResponseWriter, req *http.Request, code int) {
 	prepareNoCache(rw)
-	err := p.ClearSessionCookie(rw, req)
-	if err != nil {
-		logger.Printf("Error clearing session cookie: %v", err)
-		p.ErrorPage(rw, req, http.StatusInternalServerError, err.Error())
-		return
-	}
-	rw.WriteHeader(code)
```

Comment:
Any reason you moved this to the bottom?

Reply:
Just to have the redirectUrl be checked first?

Reply:
> Just to have the redirectUrl be checked first?

Yep, because the redirectUrl is now also required for the SignInPage when clearing the session cookie fails.